docs/install/install-cave.md

```diff
@@ -9,6 +9,7 @@ CAVE is the **C**ommon **A**WIPS **V**isualization **E**nvironment that is used
 - [**Linux: 18.2.1-6**](#linux)
 - [**Windows: 18.2.1-6**](#windows)
 - [**Mac: 18.2.1-6**](#macos)
+- [**BETA Version: 20.3.2-0.1**](install-cave-beta.md)
```

Comment:
If the file name of the markdown page changes, then this link will need to be updated slightly